initfiles/bash/etc/init.d/setupPKI.in

```diff
@@ -124,4 +124,6 @@ else
     log_success_msg
 fi
 
+chown -R ${USER_NAME}:${USER_NAME} ${homePath}
```

Comment:
Did you mean ${certPath}? ${homePath} is the home directory of the user, don't think it should be touched by this script...

Reply:
yeah I think you're right, it should be touching the /home/hpcc/certificate folder (or whatever {certPath} is set to and not everything under homePath